io/module_wrt_grid_comp.F90

```diff
@@ -1003,10 +1005,22 @@ subroutine wrt_initialize_p1(wrt_comp, imp_state_write, exp_state_write, clock,
 !
 !loop over all items in the imp_state_write and count output FieldBundles
 
-      ! if (lprnt) write(0,*)'wrt_initialize_p1: before get_outfile FBCount =', FBCount
+      ! if (lprnt) then
```

Comment:
Can we remove the prints here and lines 1017-1023?

Reply:
Done